test/server/test_image_validation.py

```diff
@@ -77,7 +77,61 @@ def test_invalid_data_url():
         assert resp.status_code == 500
 
 
+def test_invalid_data_jpeg_image():
+    image = image_setup()
+    (shot_data, shot_json, user) = user_setup()
+    valid_header = "data:image/jpeg;base64,/9j/2wBDAAQDAwQDAwQEAwQFBAQ"
+    invalid_header = "data:image/jpeg;base64,BAAQ"
+    if valid_header in image:
```

Comment:
This should be an assert, so the test doesn't silently pass, without hitting the server.